src/main/java/uk/gov/hmcts/juror/api/moj/repository/THistoryCodeRepository.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.hmcts.juror.api.moj.repository;
+
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.juror.api.bureau.domain.ReadOnlyRepository;
+import uk.gov.hmcts.juror.api.juror.domain.THistoryCode;
+
+@Repository
+public interface THistoryCodeRepository extends ReadOnlyRepository<THistoryCode, String> {
```

Comment:
Might need to suppress PMD/checkstyle on interface name and class name

Reply:
Seems to have made it through :)